app/src/main/java/com/github/dedis/student20_pop/ui/OrganizerFragment.java

```diff
@@ -137,8 +129,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
 
         //Display Events
         expandableListView = rootView.findViewById(R.id.organizer_expandable_list_view);
-        listViewEventAdapter = new OrganizerExpandableListViewEventAdapter(this.getActivity(), events);
-        expandableListView.setAdapter(listViewEventAdapter);
+        listViewEventAdapter = new OrganizerExpandableListViewEventAdapter(this.getActivity(), events, onEventTypeSelectedListener);
+        expandableListView.setAdapter((ExpandableListAdapter) listViewEventAdapter);
```

Comment:
Is it possible to change the type of listViewEventAdapter so that the cast is not needed ?

Reply:
The cast is not needed